src/cmd/flux-kvs.c

```diff
@@ -96,6 +96,18 @@ static struct optparse_option get_opts[] =  {
     { .name = "at", .key = 'a', .has_arg = 1,
       .usage = "Lookup relative to RFC 11 snapshot reference",
     },
+    { .name = "order", .key = 'o', .has_arg = 1,
```

Comment:
I don't think ```has_arg``` needs to be 1?

Reply:
Also, just a minor suggestion to use `-s, --sort` since many other tools do so

Reply:
I'm not sure ```sort``` is the right word.  What we're looking for is a word for "output one key at a time".  Perhaps ```order``` is not the best, but not sure what would be better.

Reply:
Oops - thanks for pointing out the has_arg problem.

The `--order` option ensures that values are printed in the same order as specified on the command line, as opposed to random order if they are allowed to be fetched in parallel.   Maybe `--serial-lookup` would be a better option name?

Reply:
Ah, sorry! I was confused by the usage message
```
      .usage = "For multiple keys, ensure values are output in order",
```

I still don't understand what the option does exactly, but you are definitely right `--sort` isn't correct. Sorry for the noise. (On another note, did the `flux-kvs(1)` manpage need an update here?

Reply:
The code was changed to be asynchronous, so without the option, keys could be output in any order.   Maybe I should just make this the default if `--watch` is not specified and we can avoid this option entirely?

Reply:
I think that's what I should do.  Shaving off a few milliseconds for a multiple-key get that is rarely used is not worth much, while making the command confusing does harm.

Reply:
Sounds good to me, thanks!